packages/gatsby/cache-dir/static-entry.js

```diff
@@ -57,8 +57,8 @@ export default (pagePath, callback) => {
 
   // If enabled, group files in their respective folders otherwise generated
   // non-static files will be left in the root of the output directory.
-  const pathToGroupStyles = __GROUP_FILES__ ? `styles/` : ``
-  const pathToGroupScripts = __GROUP_FILES__ ? `scripts/` : ``
+  const pathToGroupStyles = __GROUP_FILES__ ? `css/` : ``
```

Comment:
@KyleAMathews renamed to `css` and `js`

### Here's a preview build I'm working off
https://narative-hello-world.netlify.com/

### Here's the output now
![screen shot 2018-07-06 at 10 18 33 am](https://user-images.githubusercontent.com/8007686/42392046-487fe374-8106-11e8-8630-6b6a541b1b28.png)